.travis.yml

```diff
@@ -48,7 +48,7 @@ jobs:
       script: travis_retry mvn test
     - stage: release
       name: "Push to Nexus"
-      if: fork = false AND (branch =~ /support\/.*/) AND type != pull_request AND commit_message !~ /\[no-release\]/
+      if: fork = false AND (branch =~ /support\/.*/ AND type != pull_request AND commit_message !~ /\[no-release\]/
```

Comment:
I think there is a missing closing parenthesis after branch =~ /support\/.*/

Reply:
Fixed :)
Thanks.